$MirOS$

	struct tm.tm_year is time_t is 64bit is not int

--- libstdc++-v3/include/bits/locale_facets_nonio.tcc.orig	Thu Apr  9 23:22:43 2009
+++ libstdc++-v3/include/bits/locale_facets_nonio.tcc	Sun Dec  1 20:20:22 2013
@@ -764,8 +764,10 @@ _GLIBCXX_END_LDBL_NAMESPACE
 		case 'y':
 		case 'C': // C99
 		  // Two digit year. [tm_year]
-		  __beg = _M_extract_num(__beg, __end, __tm->tm_year, 0, 99, 2,
+		  __beg = _M_extract_num(__beg, __end, __mem, 0, 99, 2,
 					 __io, __tmperr);
+		  if (!__tmperr)
+		    __tm->tm_year = __mem;
 		  break;
 		case 'Y':
 		  // Year [1900). [tm_year]
